src/main/java/io/reactivex/rxjava3/annotations/NonNull.java

```diff
@@ -19,10 +19,10 @@
 import java.lang.annotation.*;
 
 /**
- * Indicates that a field/parameter/variable/return type is never null.
```

Comment:
Do we want to mention type use too?

Reply:
It's not that important; type parameter should imply its use too.